api/python/provisioner/inputs.py

```diff
@@ -686,6 +686,14 @@ class PrivateDataNetwork(ParamGroupInputBase):
         _param_group, descr="node data network mtu",
         default=1500
```

Comment:
The default value of MTU for hardware should be `9000`, else it should be `1500`